src/compile.ts

```diff
@@ -608,6 +645,16 @@ export function compile(
             ),
             toExpr(node.operand, scope),
           ]);
+        } else if (ts.isPostfixUnaryExpression(node)) {
+          return newExpr("UnaryPostfixExpr", [
```

Comment:
Should we mirror the name? PostfixUnaryExpr

Reply:
Sure